components/script/microtask.rs

```diff
@@ -38,6 +43,16 @@ pub struct EnqueuedPromiseCallback {
     pub pipeline: PipelineId,
 }
 
+/// A mutation callback
+#[derive(JSTraceable, HeapSizeOf)]
+pub struct EnqueuedMutationCallback {
+//    #[ignore_heap_size_of = "Rc has unclear ownership"]
+//    pub callback: Rc<MutationCallback>,
+    pub pipeline: PipelineId,
+//    pub mutations: Vec<Root<MutationRecord>>,
+//    pub observer: MutationObserver,
```

Comment:
What's going on with these commented out members and the commented out code that uses them?